aead-stream/Cargo.toml

```
@@ -0,0 +1,19 @@
+[package]
+name = "aead-stream"
+version = "0.1.0"
```

Would suggest syncing the version with `aead` so it's easy to know which versions are compatible

```suggestion
version = "0.6.0-pre"
```